src/lib/conf/tsconfig.ngc.json

```diff
@@ -24,7 +23,12 @@
     "lib": [
       "dom",
       "es2015"
-    ]
+    ],
+    "paths": {
+      "*": [
```

Comment:
Why is that wild-card mapping?

Reply:
Can you post me a link how CLI does implement this?

I just found this doc:; https://github.com/angular/angular-cli/blob/1156befb930faf6458040bb234d0b46b11dff21e/docs/documentation/stories/linked-library.md

Reply:
Actually I cannot find this any more, Though what I found is a test  https://github.com/angular/angular-cli/blob/master/tests/e2e/tests/build/ts-paths.ts#L31

This works slightly different, but same concept. Try to resolve the module provided using the standard resolution and if not resolved use `app/shared/*'`